bokeh/charts/_charts.py

```diff
@@ -144,17 +147,28 @@ def start_plot(self, xgrid, ygrid):
         if ygrid:
             self.make_grid(1, yaxis.ticker)
 
-        # Add tools
+        # Add tools if supposed to
         if self.tools:
+            # only create pan and zoom tools if chart is not categorical...
+            if self.categorical:
+                tools_to_remove = ['pan', 'zoom']
+            else:
+                tools_to_remove = []
```

Comment:
> It would also be easier to test this function in a unit test too, since it is returning the list of tools objects and not modifying one of its inputs.

+1